rtf

```diff
@@ -613,6 +613,11 @@ class Platform:
     self.hosts = {}
     self.plugins = {}
     self.provider = "virtualbox"
+
+    # Initialized at setup time
+    script_dir = os.path.dirname(os.path.realpath(__file__))
+    client_path = script_dir + "/rudder-api-client"
```

Comment:
why not directly put this ?
self.client_path = script_dir + "/rudder-api-client"